graylog2-web-interface/src/components/indices/IndexSummary.jsx

```diff
@@ -9,28 +11,37 @@ const IndexSummary = React.createClass({
     index: React.PropTypes.object.isRequired,
     indexRange: React.PropTypes.object.isRequired,
     isDeflector: React.PropTypes.bool.isRequired,
+    name: React.PropTypes.string.isRequired,
   },
   getInitialState() {
     return { showDetails: this.props.isDeflector };
   },
-  _deflectorBadge() {
-    if (this.props.isDeflector) {
-      return <i className="fa fa-bolt" title="Write-active index"/>;
+  _formatLabels(index) {
+    const labels = [];
+    if (index.is_deflector) {
+      labels.push(<Label key={`${this.props.name}-deflector-label`} bsStyle="primary">deflector</Label>);
     }
-  },
-  _reopenedBadged(index) {
-    if (index.reopened) {
-      return <i className="fa fa-retweet" title="Reopened index"/>;
+
+    if (index.is_closed) {
+      labels.push(<Label key={`${this.props.name}-closed-label`} bsStyle="warning">closed</Label>);
+    }
+
+    if (index.is_reopened) {
```

Comment:
Probably it shouldn't, but it does:
```
"graylog2_0": {
  "size":null,
  "range":null,
  "is_closed":true,
  "is_reopened":true,
  "is_deflector":false
}
```

<img width="1212" alt="screen shot 2016-04-04 at 14 39 10" src="https://cloud.githubusercontent.com/assets/716185/14248177/60ba7010-fa73-11e5-86a5-1c53bd407aac.png">